### PR TITLE
PR Template

### PR DESCRIPTION
**Pull request by IChocked**
# Description
The following file is a PR template for this repo. This gives us a standard way to document proposed changes to the repo and make reviewing PRs easier. WIth this change, every time you open a PR it will fill the 'description' part of the PR documentation with the markdown text written in `.github/pull_request_template.md` which is what you are viewing now.

For more information on pull request templates or how it was implemented, view the Github docs [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
The only tests done on this change was to:
1. Fork the EV repo
2. Create `.github/pull_request_template.md` in an new branch
3. Merge the branch to main on the fork
4. Make another generic PR and make sure that it fills the description with the template

See verification of that on my fork [here](https://github.com/IChocked/EV) and feel free to make test pull requests to verify this on the fork

# Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if relevant)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Description
+Please include a summary of the change and which issue is fixed or what feature is added. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Type of change
```
Comment (anaymehta) [inline .github/pull_request_template.md]: Just noticed this header is inconsistent with the rest, not sure if that was intentional or not but just a cosmetic comment

**Review by anaymehta — APPROVED** [review]
Huge fan of this, looks great to me!

**Review by ashvinnagarajan — APPROVED** [review]
Step 1 to a more robust software development pipeline! Approved!

**Review by marcuscalderon — APPROVED** [review]

**Comment by anaymehta** [comment]
Feel free to merge whenever you're ready!

**Pull request merged**